_routes/posts.md

```diff
@@ -52,6 +52,8 @@ Mentioned people will be included in a separate array for advanced displaying. A
 }
 ~~~
 
+The mention syntax in the post body above is the primary form you will see, but there's also a deprecated extension to it in the form of `@{Bobby; bob@example.com}`. The part of before the semicolon (`;`) may used as display text for the mention, while the optional `name` from the entry in in `mentioned_people` matching the diaspora ID after the semicolon is to be preferred.
```

Comment:
Extension sounds like the older version is better, which it isn't, so

```patch
-deprecated extension
+deprecated variant
```
and to make it a bit clearer

```patch
-is to be preferred
+should be used instead if available
```

maybe?

Reply:
Sounds good.